packages/commerce-sdk-react/src/provider.tsx

```diff
@@ -17,8 +20,33 @@ export interface CommerceAPIProviderProps {
     currency: string
 }
 
+export const CommerceAPIContext = React.createContext({} as ApiClients)
```

Comment:
TIL `createContext` requires a default value to be passed in. So in Typescript, we can pass in an empty object and casting the type with `as` like that.